package.sln

```diff
@@ -2,9 +2,9 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 15
 VisualStudioVersion = 15.0.27004.2005
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Unity.Mvc", "src\Unity.Mvc.csproj", "{EECD2BD0-11FC-4A97-8187-E2CA7A744CC4}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Unity.AspNet.Mvc", "src\Unity.AspNet.Mvc.csproj", "{EECD2BD0-11FC-4A97-8187-E2CA7A744CC4}"
```

Comment:
Alessandro, I would like to keep project names consistent with package names. This package was released in 2013 and since than more than 1 million people downloaded it. Changing package name would be undesirable.

Reply:
I understand your concern but Unity 5.x has already changed alot of namespaces and this one is really minor. Moreover this was just to align it with the WebApi package which is Unity.AspNet.WebApi. A typical usage of this package would mean to refer to it only in the UnityConfig.cs file which is added to the project by the NuGet package itself.